### PR TITLE
Use FRACMASK macro

### PR DESCRIPTION
**Pull request by ceski-1**

### CODE REVIEW

File: src/hu_coordinates.c [hunk 1]
```diff
@@ -71,12 +71,12 @@ static split_fixed_t SplitFixed(fixed_t x)
...
     result.negative = x < 0;
     result.base = x >> FRACBITS;
-    result.frac = x & (FRACUNIT - 1);
+    result.frac = x & FRACMASK;
 
     if (result.negative && result.frac)
     {
         result.base++;
-        result.frac = FRACUNIT - result.frac;
+        result.frac = FRACMASK - result.frac + 1;
```
Comment (fabiangreffrath) [inline src/hu_coordinates.c:79]: I think this was already correct before.

Comment (ceski-1) [reply src/hu_coordinates.c]: Fixed here: https://github.com/fabiangreffrath/woof/pull/1763/commits/77526e0e64b3ecfcb1584d2a382c220ee1d9c67b

File: src/p_extnodes.c [hunk 2]
```diff
@@ -450,7 +450,7 @@ static void P_LoadSegs_XGLN(byte *data)
             line = (unsigned short)SHORT(ml->linedef);
             side = ml->side;
 
-            if (line == 0xffff)
+            if (line == FRACMASK)
```
[the remark this answers is not in the document]
Comment (ceski-1) [reply src/p_extnodes.c]: Okay, I'll leave this one alone. Fixed here: https://github.com/fabiangreffrath/woof/pull/1763/commits/77526e0e64b3ecfcb1584d2a382c220ee1d9c67b

**Review by fabiangreffrath — APPROVED** [review]
Thank you!

**Pull request merged**